node/node.go

```diff
@@ -378,13 +377,18 @@ func makeNode(
 		eventSinks: eventSinks,
 
 		services: []service.Service{
+			// the order of these services control
```

Comment:
This comment makes me simultaneously very happy (that it's here) and very sad (that it's necessary). 😆

Reply:
extremely the same